packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.html

```diff
@@ -0,0 +1,111 @@
+<h1 id="dedicatedCloud_datacenter_backup_header" data-translate="dedicatedCloud_datacenter_backup_title"></h1>
+<div data-ui-view>
+    <div class="datacenter-backup">
+        <div data-ovh-alert="{{ ::$ctrl.alertMain }}"></div>
+        <oui-switch data-model="true" data-on-change="$ctrl.deactivate()">
+        </oui-switch>
+        <span data-translate="dedicatedCloud_datacenter_backup_enabled"></span>
+
+        <section class="datacenter-backup-offers">
+            <h4
+                class="mb-3"
+                data-translate="dedicatedCloud_datacenter_backup_choose_offers"
+            ></h4>
+            <dedicated-cloud-datacenter-backup-offers
+                data-backup-offers="$ctrl.backupOffers"
+                data-backup="$ctrl.backup"
+                data-preview-mode="false"
+                data-user="$ctrl.currentUser"
+            ></dedicated-cloud-datacenter-backup-offers>
+            <oui-button data-variant="primary">
```

Comment:
Nothing happens on clicking on the button ?

Reply:
This is being implemented still :)

Reply:
The upgrade offers feature has been implemented and merged. Thanks